### ISSUE TITLE
Users should get SSL verification errors

### ISSUE THREAD

**Issue by jason-o-matic** [DocRaptor/docraptor-php#8]
Some PHP installations don't have root CAs included, so a default configuration of the PHP package won't work because under the hood curl fails to verify DocRaptor's cert.

This will show up in the apache logs:

```
SSL certificate problem: unable to get local issuer certificate
```

Unfortunately the exception that is raised is:

```
Uncaught exception 'DocRaptor\ApiException' with message 'API call to https://docraptor.com//docs timed out
```

which is misleading.

We might be able to bundle the necessary certs with the package to solve this.

**Comment by jason-o-matic** [comment]
More info: http://unitstep.net/blog/2009/05/05/using-curl-in-php-to-access-https-ssltls-protected-sites/

**Comment by jason-o-matic** [comment]
In the meantime this can be added to work around the issue:

```
$configuration->setSSLVerification(false);
```

**Comment by AlexOverbeck** [comment]
@jason-o-matic The newest version of our package _(currently [v2.0.0](https://packagist.org/packages/docraptor/docraptor))_ now uses [guzzle](https://github.com/guzzle/guzzle) to make http requests. I did some basic testing and I believe this issue is resolved in version 2.0.0.

Given the following example:
```php
try {
  $docraptor = new DocRaptor\DocApi();
  $docraptor->getConfig()->setUsername("YOUR_API_KEY_HERE");

  $doc = new DocRaptor\Doc();
  $doc->setName("php-sync.pdf");
  $doc->setTest(true);
  $doc->setDocumentType("pdf");
  $doc->setDocumentContent("<html><body>Hello from PHP</body></html>");
  $docraptor->createDoc($doc);
} catch (DocRaptor\ApiException $exception) {
  echo $exception;
}
```
In a world where DocRaptor's ssl certificate becomes expired the above example will result in an `DocRaptor\ApiException` being caught. The output will look something like the following.
```
cURL error 60: SSL certificate problem: certificate has expired (see https://curl.haxx.se/libcurl/c/libcurl-errors.html)
```

In the source code this error originates from the DocApi class in the following location:
https://github.com/DocRaptor/docraptor-php/blob/master/lib/DocApi.php#L385-L394
